docs/source/sawtooth_cli.rst

```diff
@@ -0,0 +1,10 @@
+
+*********************************
+Sawtooth CLI
+*********************************
```

Comment:
Agreed. The terms "Reference," "Guide", "Tutorial," etc. are very useful as a way of describing the content's purpose.